src/arch/xtensa/Kconfig

```diff
@@ -8,4 +8,23 @@ config SMP
 	help
 	  Build SMP instead of UP
 
+config CORE_COUNT
+	int "Number of cores"
+	range 1 4
+	depends on CAVS
+	help
+	  Number of cores
+
+config HP_MEMORY_BANKS
+	int "HP memory  baks count"
+	depends on CAVS
+	help
+	  Avalible memory banks cont for HP memory
+
+config LP_MEMORY_BANKS
+	int "LP memory  baks count"
+	depends on CAVS
+	help
+	  Avalible memory banks cont for LP memory
```

Comment:
Diito for LP, lets say Low Power

Reply:
available again, and it's banks not baks.